pkgs/development/python-modules/firetv/default.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, adb-homeassistant
+, flask
+, pure-python-adb-homeassistant
+, pycryptodome
+, pyyaml
+, rsa
+}:
+buildPythonPackage rec {
+  pname = "firetv";
+  version = "1.0.9";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "602de77411c2caffb322e4ff63fa6cc4eeb9a50c5f4b14e13930ed7cd87cf513";
+  };
+
+  propagatedBuildInputs = [
+    adb-homeassistant
+    flask
+    pure-python-adb-homeassistant
+    pycryptodome
+    pyyaml
+    rsa
+  ];
+
+  meta = with lib; {
+    description = "Communicate with an Amazon Fire TV device via ADB over a network.";
```

```suggestion
    description = "Communicate with an Amazon Fire TV device via ADB over a network";
```
See https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#submitting-changes.

---

tbh i just used [python-package-init]( https://github.com/costrouc/nixpkgs-dev-tools/blob/master/python/python-package-init.py ) and never checked for these issues. i will add some changes to avoid a leading punctuation mark.